book-domain-service/app/db.py

```diff
@@ -0,0 +1,24 @@
+import sqlite3
+
+from config import Config
+from app import db
+from app import app
+from flask import g
+
+
+def get_db(database_path):
+
+    if 'db' not in g:
+        g.db = sqlite3.connect(
+            database_path,
+            detect_types=sqlite3.PARSE_DECLTYPES
+        )
+        g.db.row_factory = sqlite3.Row
+
+    return g.db
+
+def init_db( sql_file_path , database_path): 
+    db = get_db(database_path)
+ 
+    with app.open_resource( sql_file_path, mode='r') as file:
```

Comment:
file is a reserved python keyword so you wouldn't want to use it as a variable name. A convention with python if you want to use a reserved word is to append an _ to it so you would have with app.open_resource( sql_file_path, mode='r') as file_: